### DISCUSSION TITLE
Where can i find etcd rpms

### DISCUSSION THREAD

**Discussion by rk-arise** [etcd-io/etcd#18088] [Q&A] [answered]
Where can i find etcd rpms

**Comment by tjungblu** [comment, marked as answer]
We (as in this project) do not release RPMs. For fedora @ingvagabund maintains them here: https://src.fedoraproject.org/rpms/etcd

**Reply by ingvagabund** [reply]
The packaging work is currently maitainted by fedora's [go-sig](https://pagure.io/GoSIG/go-sig).
